src/utils.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+  /**
+   * Just like parseInt, but fixed to base-10
+   */
+  Int: function (str) {
+     return parseInt(str, 10);
+  },
+
+  /**
+   * Cycles through an object and changes all numeric fields to ints
+   * where necessary. 'context' is used for exception reporting and can be
+   * left unset upon invocation.
+   */
+  cleanJSON: function cleanJSON(json, context) {
```

Comment:
```suggestion
  cleanJSON(json, context) {
```

Reply:
This doesn't actually work, due to the recursion here:
https://github.com/iFixit/node-markup/blob/e8dada272c7e69411a1a006513cc94f095307763/src/utils.js#L30